src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const result: GroupsMap<T> = {};
+
+  items.forEach((element) => {
+    const resultKey:string = String(element[key]);
```

Comment:
не обов`язково вказувати тип, це ж видно і так, що буде строка на виході з функції

```suggestion
    const resultKey = String(element[key]);
```